opensrp-chw/src/main/java/org/smartregister/chw/util/AllClientsUtils.java

```diff
@@ -112,4 +85,65 @@ private static Intent initProfileActivityIntent(Activity activity, CommonPersonO
         intent.putExtra(org.smartregister.chw.anc.util.Constants.ANC_MEMBER_OBJECTS.TITLE_VIEW_TEXT, R.string.return_to_all_client);
         return intent;
     }
+
+    public static void goToOtherMemberProfile(Activity activity, CommonPersonObjectClient patient,
+                                              Bundle bundle, String familyHead, String primaryCaregiver) {
+
+        if (StringUtils.isBlank(familyHead) && StringUtils.isBlank(primaryCaregiver)) {
+            showShortToast(activity, activity.getString(R.string.error_opening_profile));
+        } else {
+            String registerType = patient.getDetails().get(REGISTER_TYPE);
+            Intent intent;
+            if (CoreConstants.REGISTER_TYPE.INDEPENDENT.equals(registerType)) {
+                intent = new Intent(activity, AllClientsMemberProfileActivity.class);
+            } else {
+                intent = new Intent(activity, FamilyOtherMemberProfileActivity.class);
+            }
+            intent.putExtras(bundle != null ? bundle : new Bundle());
+            intent.putExtra(Constants.INTENT_KEY.BASE_ENTITY_ID, patient.getCaseId());
+            intent.putExtra(CoreConstants.INTENT_KEY.CHILD_COMMON_PERSON, patient);
+            intent.putExtra(Constants.INTENT_KEY.FAMILY_HEAD, familyHead);
+            intent.putExtra(Constants.INTENT_KEY.PRIMARY_CAREGIVER, primaryCaregiver);
+            intent.putExtra(Constants.INTENT_KEY.VILLAGE_TOWN, patient.getDetails().get(OpdDbConstants.KEY.HOME_ADDRESS));
+            activity.startActivity(intent);
+        }
+    }
+
+    @NotNull
+    public static List<OpdEventClient> getOpdEventClients(String jsonString) {
+        List<OpdEventClient> allClientMemberEvents = new ArrayList<>();
+
+        FamilyEventClient locationDetailsEvent = org.smartregister.family.util.JsonFormUtils.processFamilyUpdateForm(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.family.util.JsonFormUtils.processFamilyUpdateForm' due to existing import 'org.smartregister.family.util.JsonFormUtils'](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5522858)